pyproject.toml

```diff
@@ -25,7 +25,8 @@ classifiers = [
 ]
 dependencies = [
     "pyarrow",
-    "rpy2>=3.3"
+    # The intent is rpy2>=3.5, but with an exception for Python 3.4.
+    "rpy2 >= 3.3, != 3.4.*"
```

Comment:
Why are you excluding rpy2 3.4.x? (that's what we were originally hoping to use).

Were you attempting to do something like: `rpy2>=3.3; python_version > "3.4"`?

Reply:
Ah. I had understood your first comment in the PR as "I need 3.3 but can't test versions between that and latest (3.5)".

Reply:
Let me revert then.

Reply:
Ah... thank you! I have tested somewhat thoroughly this with rpy2 3.4.x . I'm happy for the lower constraint to be 3.4 if you like. I just chose 3.3 because that goes far enough back (to pandas 1.0.0 days) that I doubt it makes sense to go further. You may know of some reason to exclude 3.3.x.